libs/pageserver_api/src/models.rs

```diff
@@ -155,6 +155,7 @@ pub struct TenantConfigRequest {
     pub lagging_wal_timeout: Option<String>,
     pub max_lsn_wal_lag: Option<NonZeroU64>,
     pub trace_read_requests: Option<bool>,
+    pub eviction_policy: Option<serde_value::Value>,
```

Comment:
Why is this a dom object?

Related: serde_json::Value is probably the same as serde_value.

Reply:
So, the (wrong) reasoning behind this was that a `serde_json::Value` would only deserialize JSON.
Of course that's wrong, as demonstrated in this playground:

https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=b5352d8d94a429e74dc6fe981ef6454b

However, I don't understand why the `serde` crate doesn't provide such a type.

Reply:
Oki but I think the why is that "this type is in pageserver_api and the real type is not" right? IMHO should be a comment, is the plan to expose the one type but not now?